server/service/src/invoice/supplier_return/insert/mod.rs

```diff
@@ -90,6 +97,25 @@ pub fn insert_supplier_return(
                 })?;
             }
 
+            // Update to not new status after upserting lines
+            if supplier_return.original_shipment_id.is_some() {
+                update_supplier_return(
+                    ctx,
+                    UpdateSupplierReturn {
+                        supplier_return_id: supplier_return.id.clone(),
+                        comment: None,
+                        status: Some(UpdateSupplierReturnStatus::Shipped),
+                        colour: None,
+                        on_hold: None,
+                        their_reference: None,
+                        transport_reference: None,
```

Comment:
```suggestion
                    UpdateSupplierReturn {
                        supplier_return_id: supplier_return.id.clone(),
                        status: Some(UpdateSupplierReturnStatus::Shipped),
                        ..Default::default()
```
Can default the None values instead of declaring each one